docs/source/jupyter/ontology-interface.ipynb

```diff
@@ -107,101 +162,415 @@
+    "\n",
+    "- By **label**. Fetchs the entity by the label that has been assigned to it using either the `rdfs:label` or `skos:prefLabel` predicates.\n",
+    "\n",
+    "- By **IRI**. The full [IRI](https://fusion.cs.uni-jena.de/fusion/blog/2016/11/18/iri-uri-url-urn-and-their-differences/) of an ontology entity is provided in order to fetch it.\n",
+    "\n",
+    "- By **string**. Using a string, for example `\"city.LivingBeing\"`. This is only useful in some special cases."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The **most convenient way** to access an ontology entity is using the **dot notation** in python, i.e. `city.Citizen`. This method is a shorthand for fetching by suffix or label:  \n",
+    "\n",
+    "- When the keyword `reference_by_label` is set to `True` (enabled) in the [ontology YAML installation file](../yaml.md), the dot notation is a shorthand for fetching by label. This keyword is **enabled** in the `math` namespace.\n",
+    "\n",
+    "- When the keyword `reference_by_label` is set to `False` (disabled) or not set, the dot notation is a shorthand for fetching by suffix instead. This keyword is **disabled** in the `city` namespace."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Accessing an ontology entity by suffix**\n",
+    "\n",
+    "Let's fetch the Citizen class, whose IRI is `http://www.osp-core.com/city#Citizen`."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The keyword, `reference_by_label` is set to `False`, so one can just use the dot notation. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
    "outputs": [
     {
-     "output_type": "stream",
-     "name": "stdout",
-     "text": [
-      "\nYou can use the namespace to access its entities\ncity.LivingBeing\n\nYou can also use index notation. For owl ontologies, this queries the entities by rdf.label.It returns a list of all entities with the same label\nTrue\n\nYou can access the namespace of an entity\nFalse\n"
-     ]
+     "data": {
+      "text/plain": [
+       "<OntologyClass city.Citizen>"
+      ]
+     },
+     "execution_count": 4,
+     "metadata": {},
+     "output_type": "execute_result"
     }
    ],
    "source": [
-    "from osp.core.namespaces import city\n",
-    "\n",
-    "print(\"\\nYou can use the namespace to access its entities\")\n",
-    "print(city.LivingBeing)\n",
+    "city.Citizen"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Another alternative is using the `get_from_suffix` method from the namespace object. This is useful when the suffix contains special characters."
```

Comment:
The sentence "This is useful when the suffix contains special characters" might be confusing. Perhaps it will be helpful to explain why.

Reply:
Changed to _This is useful when the suffix contains characters that Python does not accept as property names, such as spaces or dashes._